main.go

```diff
@@ -3,8 +3,10 @@ package main
 import (
```

Comment:
Yes. "we don't block the process from dying for too long". That is "we don't block [..] for too long".